src/equations/compressible_euler_2d.jl

```diff
@@ -1016,7 +1016,8 @@ end
 
 # Called inside `FluxRotated` in `numerical_fluxes.jl` so the direction
 # has been normalized prior to this rotation of the state vector
-@inline function rotate_to_x(u, normal_vector, equations::CompressibleEulerEquations2D)
+@inline function rotate_to_x(u, normal_vector,
+                             equations::Union{CompressibleEulerEquations2D, ShallowWaterEquations2D})
```

Comment:
TBH, I wouldn't reuse this. `rotate_to_x` and `rotate_from_x` are two extremely simple functions and I would just recreate them for the SWE. This way, everything SWE-related is contained in that file.

Reply:
I am fine with that. I originally had an identical version inside the shallow water file but removed it to reuse. I can undo that decision